modules/simulator.js

```diff
@@ -63,8 +63,13 @@ function step(ctx, land)
   }
 }
 
+function getRandomInt(min, max) {
```

Comment:
`Util.js` sounds good. @Youngjk could you please replace other `Math.Floor` code with this Util method?